feature/gribi/ate_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -103,3 +132,40 @@ func TestCompliance(t *testing.T) {
 		})
 	}
 }
+
+// configureDUT configures port1-3 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	p1 := dut.Port(t, "port1")
+	p2 := dut.Port(t, "port2")
+	p3 := dut.Port(t, "port3")
+
+	dut.Config().Interface(p1.Name()).Replace(t, dutPort1.NewInterface(p1.Name()))
+	dut.Config().Interface(p2.Name()).Replace(t, dutPort2.NewInterface(p2.Name()))
+	dut.Config().Interface(p3.Name()).Replace(t, dutPort3.NewInterface(p3.Name()))
+
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(*nonDefaultNI)
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static")
```

Comment:
Is "static" vendor independent?

Reply:
I believe different implementations have different requirements here, but that is a more general issue we need to sort out still.  I think strictly speaking any value can be used here, but that seems like it has caused some problems.

This will likely need to change to a deviation in the future once we sort out what it _should_ be.